lib/src/pub_updater.dart

```diff
@@ -36,6 +37,15 @@ class PubUpdater {
   }) async {
     final latestVersion = await getLatestVersion(packageName);
 
+    final currentVersionDesc = Version.parse(currentVersion);
+    final latestVersionDesc = Version.parse(latestVersion);
+
+    if (!latestVersionDesc.isPreRelease && currentVersionDesc.isPreRelease) {
+      // If there is a stable version available, pre release versions are not
+      // considered out of date
```

Comment:
```suggestion
      // If there is a stable version available, pre-release versions are not
      // considered out of date.
```

Missing full stop. In addition, I think "pre release" is meant to be "pre-release". At least [here](https://news.dartlang.org/2017/09/dart-20-pre-releases-what-they-mean-to.html) they are hyphenating it.

Reply:
They are using hyphen here.
https://semver.org/spec/v2.0.0-rc.1.html#spec-item-10

Reply:
Indeed, that's why I think we should use the hyphen.

Reply:
changing to [prerelease](https://dart.dev/tools/pub/publishing#publishing-prereleases)

Reply:
That works for me to. I'm surprised by the inconsistency in different place.